todo_app/test_system.py

```diff
@@ -46,26 +32,25 @@ def test_app():
 
     # Tear Down     
     thread.join(1)  
-    delete_board(board_id) 
+    test_task = db.get_task("TestItem")
+    if test_task is not None:
+        id = test_task["_id"]        
+        db.delete_task(id)
```

Comment:
This is a bit fragile/doesn't scale:
It depends on "TestItem" staying in sync with any tests. What if there are tests that set up multiple items?